demosplan/DemosPlanCoreBundle/Logic/FilterUiDataProvider.php

```diff
@@ -25,7 +25,7 @@ class FilterUiDataProvider
 
     public function __construct()
     {
-        $this->relativeFilterNamesPath = 'demosplan/addons/workflow/SegmentsManager/Resources/config/segmentsFilterNames.yaml';
+        $this->relativeFilterNamesPath = 'demosplan/DemosPlanCoreBundle/Resources/config/segmentsFilterNames.yaml';
```

Comment:
This looks like something that in time wants to be moved to a config loader / compiler pass.

Reply:
true, this should be some part of a configuration. I'll try to keep that in mind as well for future refactorings.